ftw/linkchecker/link_accumulation.py

```diff
@@ -61,64 +62,75 @@ def _separate_links_to_internal_external(self):
             elif not link_obj.is_internal:
                 self._external_link_objs.append(link_obj)
 
-    def _find_links_on_brain_fields(self, brain):
+
+class LinkOnFieldSeeker(object):
```

Comment:
The name of the class is hard to grasp for me.

A docstring would really be helpful in order to understand what a LinkOnFieldSeeker does: 
- Does it extract links from brains?
- And where does the word "field" from the name of the class come into play?

Reply:
True point, thanks for mentioning. I tried to summarize the steps done in a docstring (in c9d9920). Hopefully, that helps to find the code parts better responsible for each step. I was thinking of also referencing and documenting each method but I'm not sure if it improves the code really. What do you think?

Reply:
b928a08 unternimmt einen weiteren Versuch etwas besser zu erklären.